### ISSUE TITLE
Where are the Clojure communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/clojure#231]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Clojure supported by one or more large organizations?
* Does Clojure have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Clojure? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Clojure? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Clojure? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Clojure?
* Are there popular newsletters for Clojure?
* Is Clojure taught at programming bootcamps? (If so, what are some examples?)
* Is Clojure taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Clojure?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by CrowsVeldt** [comment]
The first one is easy. Clojure is supported by [Cognitect](https://cognitect.com/).

Communities online include [r/Clojure](https://www.reddit.com/r/Clojure/), and there's a list [on the Clojure website](https://clojure.org/community/resources#_clojure_discussion). I'm not personally familiar with the communities listed, so maybe someone who is could add some info on how active they are.

The same page has [a list of Clojure conferences](https://clojure.org/community/resources#_conferences). The big ones are Clojure/conj, Clojure/west, and EuroClojure.
I haven't personally had the chance to go to one yet, but I hear that they're generally packed.

[ClojureBridge](http://www.clojurebridge.org/) is all about putting together events for introducing Clojure to women, and underrepresented groups in tech.

The Clojure site also has a list of newsletters including [the Clojure Gazzette](https://purelyfunctional.tv/newsletter/), [Clojure Weekly](http://reborg.tumblr.com/), and [the REPL](https://therepl.net/). I wasn't familiar with them either, so this thread has already helped someone!

**Comment by kytrinyx** [comment]
Sweet, thanks! This is great information.

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌼

**Comment by souenzzo** [comment]
No mention to slack community?
http://clojurians.net/
I think that slack is the main community hub. The main authors of the main libraries are there (include clojure core team) ready to talk

**Comment by CrowsVeldt** [comment]
I'm still very new with Clojure, and even newer at interacting with the community. I guess I just forgot about slack. :man_shrugging:
